wallet/src/de/schildbach/wallet/request/AddCreateReq.java

```diff
@@ -0,0 +1,191 @@
+package de.schildbach.wallet.request;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by ABC on 7/26/2017.
+ */
+
+public class AddCreateReq {
+
+
+    /**
+     * phone : 9414477260
+     * email : demo@geni.to
```

Comment:
remove this domain. However, you can use @wallofcoins.com or @genitrust.com

Reply:
@rgenito I have removed all domain with @geni.to.